doc/results.Rmd

```diff
@@ -302,15 +306,31 @@ data %>%
   add_footer_lines(values = "none of those abbreviations are in the table but they demonstrate how to include abbreviations using my helper functions")
 
 ```
+\newpage
+
+## Table 3: Characteristics for penguin species subgroups
+```{r}
 
+footer_tbl_val <- "Table values are n (%) or median (25th, 75th percentile)"
+
+data %>% 
+  infuse_dictionary(meta, units = 'descriptive') %>% 
+  tbl_summary(by = 'species', missing = 'no') %>% 
+  modify_footnote(c(all_stat_cols()) ~ NA) %>% 
+  as_flex_table() %>% 
+  theme_box() %>% 
+  add_footer_lines(footer_tbl_val) %>% 
+  align(j = c(2,3,4), align = 'center', part = 'all')
+
+
+```
 
 <!-- new page not needed at the end of a block section -->
 
-`r page_long_above()`
 
 To make a landscape section, use the block_section code here. Note that the section will be applied to all content prior to the block reaching up to the most recent block.
 
-`r page_wide_above()`
+
```

Comment:
Deleting this causes an issue in the document. The issue is that we have text describing how to show results in a landscape layout, but then we don't show any results in a landscape layout. Another issue is that the text about landscape results looks like it's a footnote for Table 3. 

![image](https://github.com/user-attachments/assets/f7e0c12d-6b5d-439b-9e97-0447bdae1c55)